packages/libp2p/src/dcutr/dcutr.ts

```diff
@@ -117,21 +113,73 @@ export class DefaultDCUtRService implements Startable {
   }
 
   /**
-   * Perform the inbound connection upgrade as B
+   * This is the callback function for the onConnect handler of {@link multicodec} connections.
+   * We make sure that the connection is an inbound relayed connection and then initiate the upgrade.
    *
-   * @see https://github.com/libp2p/specs/blob/master/relay/DCUtR.md#the-protocol
+   * The protocol starts with the completion of a relay connection from A to B.
+   *
+   * @param {PeerId} peerId
+   * @param {Connection} connection
+   * @returns
    */
-  async upgradeInbound (relayedConnection: Connection): Promise<void> {
+  async onConnectAttempt (peerId: PeerId, connection: Connection): Promise<void> {
+    debugLog(`onConnectAttempt ${connection.direction === 'inbound' ? 'from' : 'to'} ${peerId}`)
```

Comment:
Please do not use template strings in logging.

If logging is disabled, any logging should be a no-op - with template strings code is executed.

Instead use formatters.

E.g.

```suggestion
    log,trace('onConnectAttempt %s %p', connection.direction, peerId)
```

Reply:
https://github.com/libp2p/js-libp2p/pull/1932/commits/65b6b7352dcc4e0bdaf0f6776cd58f30b5c61b0c